icu4c/source/data/buildtool/utils.py

```diff
@@ -86,6 +86,14 @@ def get_all_output_files(requests, include_tmp=False):
     return [f for _, f in set((type(f), f) for f in files)]
 
 
+def compute_directories(requests):
+    dirs = set()
+    for file in get_all_output_files(requests, include_tmp=True):
+        path = "%s/%s" % (dir_for(file), file.filename)
```

Comment:
I don't remember... is it ok to hardcode the `/` here because you convert it to the platform separator elsewhere?

Reply:
That's correct.  / to \ happens at the boundary.